books/RayTracingTheRestOfYourLife.html

```diff
@@ -47,22 +47,51 @@
 
 
 
-
 A Simple Monte Carlo Program
 ====================================================================================================
+<div class='together'>
+Let’s start with one of the simplest Monte Carlo programs. If you're not familiar with Monte Carlo
+programs, then it'll be good to pause and catch you up. There are two kinds of randomized
+algorithms: Monte Carlo algorithms and Las Vegas algorithms. Randomized algorithms can be found
```

Comment:
“There are two kinds of randomized algorithms: Monte Carlo ~algorithms~ and Las Vegas ~algorithms~.”